client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-status-content/index.tsx

```diff
@@ -285,7 +293,28 @@ export default function SiteStatusContent( {
 					</div>
 				);
 			}
-			return <div></div>;
+
+			if ( hasBoost ) {
+				return (
+					<a
+						className="sites-overview__column-action-button"
+						href={ adminUrl }
```

Comment:
I believe eventually we'll need to add an explanation in the dashboard that this link is redirecting to wp-admin but for now could we update the link to redirect to the add Boost page `wp-admin/admin.php?page=my-jetpack#/add-boost`

Reply:
@oikeme, thanks for pointing this out. I have updated the link to the **'Add boost'** page. For the WPCOM atomic site, in case they end up with the configure button. I redirected them to `/wp-admin/admin.php?page=jetpack` since they didn't seem to have the 'My Jetpack' module and **'Add boost'** page.